pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -645,6 +642,10 @@ func (m *cgroupManagerImpl) Create(cgroupConfig *CgroupConfig) error {
 		return err
 	}
 
+	if err := manager.Set(libcontainerCgroupConfig.Resources); err != nil {
+		return err
+	}
+
```

Comment:
m.Update() below already call `Set` so apparently there's no need to call it explicitly.